### PR TITLE
Add branding setting

### PR DESCRIPTION
**Pull request by jakubonty**
Added branding setting, which contains banner url of a channel.

### CODE REVIEW

**Comment by FunkyloverOne** [comment]
Why is this not yet merged?

**Comment by flvrone** [comment]
@jakubonty I suspect that your PR will not be merged without specs. Could you provide them please? :D

**Comment by dgb** [comment]
Hi there,

My name is Derek and I'm now one of the maintainers of the Yt gem. I want to both thank you for your contribution and apologize for the lack of feedback on your pull request. Maintaining an open source project like Yt is a pretty big job, and we're taking steps to prioritize maintenance and development of the project. We're hoping to review community contributions on a regular basis, as well as provide a roadmap for our development plans.

As part of this process we're going through the open PRs and taking action on each one, and for this PR we've chosen to review it. You can expect some more specific feedback in the next week or so.

Thanks again for your contribution, and please let us know if you have any questions or comments.

File: lib/yt/collections/branding_settings.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      # @see https://developers.google.com/youtube/v3/docs/channels#brandingSettings
+      def list_params
+        super.tap do |params|
+          params[:params] = branding_settings_params
+          params[:path] = '/youtube/v3/channels'
+        end
+      end
+
+      def branding_settings_params
+        params = {max_results: 50, part: 'brandingSettings'}
```
Comment (dgb) [inline lib/yt/collections/branding_settings.rb:26]: I think `max_results` is configurable in other requests--we should mimic that (or omit it if its irrelevant in this particular case).

**Review by dgb — CHANGES_REQUESTED** [review]
I'm 👍 on adding this feature so long as tests are added; It can probably just be a couple cases in the `as_account` channel spec, for reading/writing this attribute.

**Comment by flvrone** [comment]
Hey @dgb, could you please give some more assistance on this one? 
I really need this feature for years now, but I do not have any idea how this gem actually works inside and how to test it at all :c

Here's something my colleague tried:
https://github.com/LinkUpStudioUA/yt/commit/d4f605b014fec224610b6db8d66b8f3e39fc1807

**Comment by kangkyu** [comment]
Thank you for a really good pull request and it will be nice if we have brandingSettings but `brandingSettings.image` is deprecated from YouTube for now. Closing

**Pull request closed without merging**